.travis.yml

```diff
@@ -17,15 +16,18 @@ env:
 
 matrix:
   fast_finish: true
-  allow_failures:
-    - os: osx
   include:
-    - python: 2.7
+    - sudo: required
+      python: 2.7
       env: ENV=lint
       before_script: TOX_ENV=lint
-    - python: 2.7
+    - sudo: required
+      python: 2.7
       env: ENV=docs
       before_script: TOX_ENV=docs-travis
+      before_install:
+        - sudo apt-get update -qq
```

Comment:
Sadly `addons` stuff doesn't appear to work (I didn't dig in too much), but at least "jobs getting stuck" part is fixed now.

I believe builds got stuck because "sudo: required" is not a valid syntax or something related to that.